diff -ru orig/etc/arcacmd-scanner.conf new/etc/arcacmd-scanner.conf
--- orig/etc/arcacmd-scanner.conf	2009-01-15 10:11:44.000000000 +0100
+++ new/etc/arcacmd-scanner.conf	2009-05-09 14:22:43.609786250 +0200
@@ -7,7 +7,7 @@
 # page.
 
 # [-bases] Bases path (must be slash-terminated)
-#-Path-Bases /var/cache/arcabit/bases/
+-Path-Bases /var/lib/arcavir/bases/
 
 # [-hosts] 0 - all, 1 - only masks from hmask
 #-ScanFiles 0
Only in new/etc: arcacmd-scanner.conf~
diff -ru orig/etc/arcad-scanner.conf new/etc/arcad-scanner.conf
--- orig/etc/arcad-scanner.conf	2009-01-15 10:12:19.000000000 +0100
+++ new/etc/arcad-scanner.conf	2009-05-09 14:22:52.903117813 +0200
@@ -2,7 +2,7 @@
 # Copyright (C) 2006-2008 ArcaBit Sp. z o.o.
 
 # [-bases] Bases path (must be slash-terminated)
-#-Path-Bases /var/cache/arcabit/bases/
+-Path-Bases /var/lib/arcavir/bases/
 
 # [-hosts] 0 - all, 1 - only masks from hmask
 #-ScanFiles 0
Only in new/etc: arcad-scanner.conf~
diff -ru orig/etc/arcascanner-default.conf new/etc/arcascanner-default.conf
--- orig/etc/arcascanner-default.conf	2009-01-15 10:11:05.000000000 +0100
+++ new/etc/arcascanner-default.conf	2009-05-09 14:22:28.456489653 +0200
@@ -7,7 +7,7 @@
 # page.
 
 # [-bases] Bases path (must be slash-terminated)
--Path-Bases /var/cache/arcabit/bases/
+-Path-Bases /var/lib/arcavir/bases/
 
 # [-hosts] 0 - all, 1 - only masks from hmask
 -ScanFiles 0
Only in new/etc: arcascanner-default.conf~
diff -ru orig/usr/bin/arcaupdate new/usr/bin/arcaupdate
--- orig/usr/bin/arcaupdate	2009-03-04 13:58:17.000000000 +0100
+++ new/usr/bin/arcaupdate	2009-05-09 14:11:45.276530894 +0200
@@ -38,8 +38,8 @@
 
 # directory with anti-virus databases
 # ARCAVIR_BASES=/usr/local/bin/arcavir_bases
-# ARCAVIR_BASES=/var/lib/arcavir/bases
-ARCAVIR_BASES=/var/cache/arcabit/bases
+ARCAVIR_BASES=/var/lib/arcavir/bases
+# ARCAVIR_BASES=/var/cache/arcabit/bases
 
 # path to PGP with public key
 # comment out if you do not want verify data
Only in new/usr/bin: arcaupdate~
